consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/network/ValidatorPeers.java

```diff
@@ -84,7 +89,10 @@ private void sendMessageToSpecificAddresses(
               try {
                 connection.sendForProtocol(PROTOCOL_NAME, message);
               } catch (final PeerNotConnected peerNotConnected) {
-                LOG.trace("Lost connection to a validator.");
+                LOG.trace(
+                    "Lost connection to a validator. remoteAddress={} peerInfo={}",
```

Comment:
If we want to know when we gain/lose validators, we should log that in the add/remove? An error here means we literally ran into the someone disconnected between creating the stream and getting to their turn.